src/builder.rs

```diff
@@ -342,7 +338,7 @@ impl OutputInfo {
         let fvk: FullViewingKey = (&SpendingKey::random(rng)).into();
         let recipient = fvk.address_at(0u32, Scope::External);
 
-        Self::new(None, recipient, NoteValue::zero(), None)
+        Self::new(None, recipient, NoteValue::zero(), [0u8; 512])
```

Comment:
This technically changes the behaviour of this function, although not in a way that I think matters to callers.

Reply:
It's not a pure refactoring, strictly speaking; it is a behavior change. The alternative here would be to just inject random data? But since this then ends up encrypted to a random recipient, I don't think it matters.